src/App.tsx

```diff
@@ -15,7 +15,10 @@ import Week2 from './components/Home/Ritika/Week2';
 import RWeek3 from './components/Home/Ritika/Week3';
 import { ToastContainer } from 'react-toastify';
 import 'react-toastify/dist/ReactToastify.css';
+import AdityaWeek4Week1 from './components/Home/Aditya/Week4/pages/Week1';
+import AdityaWeek4Week3 from './components/Home/Aditya/Week4/pages/Week3';
```

Comment:
try redirecting to these "AdityaWeek4Week1" & "AdityaWeek4Week3" components, from your "AdityaWeek4" component itself and try avoiding those direct imports in App.tsx, otherwise App.tsx would contain large line of codes.

Reply:
I had tried to do that before also, however it was throwing 404 errors for accessing those routes without these imports